draft-ietf-tls-subcerts.md

```diff
@@ -491,14 +492,13 @@ On receiving a delegated credential and a certificate chain, the peer
 validates the certificate chain and matches the end-entity certificate to the
 peer's expected identity.  It also takes the following steps:
 
-1. Validate that DelegatedCredential.cred.valid_time is no more than 7 days.
 1. Verify that the current time is within the validity interval of the credential.
    This is done by asserting that the current time is no more than the
    delegation certificate's notBefore value plus DelegatedCredential.cred.valid_time.
 1. Verify that the delegated credential's remaining validity time is no more than the maximum validity
-   period. This is done by asserting that the current time is no more than the delegation
```

Comment:
Why did this need rewording?

Reply:
The previous wording was
```
This is done by asserting that the current time is no more than the delegation
certificate's notBefore value plus DelegatedCredential.cred.valid_time plus
the maximum validity period.
```
and translates to `now() <= notBefore + valid_time + 7d`, which is different from `notBefore + valid_time <= now() + 7d` (the new wording).

Reply:
Gotcha. I would consider introducing the expiry time explicitly.

Reply:
No, I don't think it translates to the expression using "now()" that you have above. As currently described, you can pre-generate a bunch of subcerts, but with this change you cannot.

Reply:
The recipient validates the subcert, not the generator, so you can still pregenerate, you just can't use them until they are valid. Am I missing something?

Reply:
I mis-read the definition of "expiry time."  I proposed a wording change that should make it more clear.  Thanks!